digits/model/images/classification/test_views.py

```diff
@@ -550,6 +564,52 @@ def test_classify_many_json(self):
         data = json.loads(rv.data)
         assert 'classifications' in data, 'invalid response'
 
+    def test_classify_one_mean_pixel(self):
+        # carry out one inference test per category in dataset
+        for category in self.imageset_paths.keys():
+            image_path = self.imageset_paths[category][0]
+            image_path = os.path.join(self.imageset_folder, image_path)
+            with open(image_path,'rb') as infile:
+                # StringIO wrapping is needed to simulate POST file upload.
+                image_upload = (StringIO(infile.read()), 'image.png')
+
+            rv = self.app.post(
+                    '/models/images/classification/classify_one?job_id=%s' % self.model_id,
+                    data = {
+                        'image_file': image_upload,
+                        'use_mean': 'pixel',
```

Comment:
I don't think this form field `use_mean` is used during inference. Instead we are using whichever mean subtraction method was used during training.
If you want to test training and inference with various mean subtraction methods perhaps you're better off creating new subclasses of `BaseTestCreated` like [there](https://github.com/NVIDIA/DIGITS/blob/master/digits/model/images/classification/test_views.py#L642)?